### ISSUE TITLE
json: cannot unmarshal number into Go struct field Message.messages.blocks of type string for GetConversationHistory

### ISSUE THREAD

**Issue by kim-barker** [slack-go/slack#1123]
### What happened
GetConversationHistory returns the following error:
```json: cannot unmarshal number into Go struct field Message.messages.blocks of type string```

### Steps to reproduce
Use conversation with date Slack block that includes timestamp
```
{
    "type": "date",
    "timestamp": 1666933200,
    "format": "{date} at {time}",
    "fallback": "not available",
    "style": {
        "bold": true
    }
},
```

#### reproducible code
```
api := slack.New("YOUR_TOKEN")
resp, err := api.GetConversationHistory(&slack.GetConversationHistoryParameters{
    ChannelID: "ChannelID",
    Inclusive: true,
    Limit:     1,
    Oldest:    "timestamp_of_message_above",
})
```

### Versions
- Go:
- slack-go/slack:

**Comment by Niraj-Fonseka** [comment]
➕ getting the same error

**Comment by kanata2** [comment]
It will be fixed in v0.12.
